### PR TITLE
feature/COM-2911

### PR DESCRIPTION
**Pull request by ysampal**
* Added filters for Account Name, status,
* Renamed submitted date to created date and updated bindings
* Added Quote Number column and filter

### CODE REVIEW

File: templates/modules/b2b-account/quotes/quotes.hyper.live [hunk 1]
```diff
@@ -28,18 +28,49 @@
...
+                </div>
+                <div class="exp-date-filter-container">
+                    <Span class="mz-b2b-quotes-compare"> > </Span>
+                    <input type="date" data-mz-is-datepicker id="expirationdate" class="expirationdate-filter" data-mz-action="dateChanged" data-format="yy/mm/dd" placeholder="Search Expiration Date">
+                </div>
+                <div class="fill-remaining-space"></div>
+                <div>
+                    <select class="status-select" data-mz-action="selectionChanged" id="statusDropdown" data-mz-value="">
+                    <option selected="selected" value="">{{ labels.filterStatus }}</option>           
+                    <option value="Pending">Pending </option>
```
Comment (Rameshdesaiise) [inline templates/modules/b2b-account/quotes/quotes.hyper.live]: Can we ready dropdown options from json files??

File: templates/modules/b2b-account/quotes/quotes.hyper.live [hunk 2]
```diff
@@ -28,18 +28,49 @@
...
+                            {% endif %}
+                        {% endfor %}
+                    </select>
+                </div>
+                {% endif %}
+                <div>
+                    <input type="text" name="searchQuoteNumber" id="searchQuoteNumber" class="quotesTextSearch" data-mz-action="inputChanged" placeholder="Quote Number">
+                </div>
+                <div>
+                    <input type="text" name="searchName" id="searchName" class="quotesTextSearch" data-mz-action="inputChanged" placeholder="Quote Name">
```
Comment (Rameshdesaiise) [inline templates/modules/b2b-account/quotes/quotes.hyper.live]: get all placeholder from json file

**Review by Rameshdesaiise — CHANGES_REQUESTED** [review]
I added minor changes

File: templates/modules/b2b-account/quotes/quotes.hyper.live (hunk 2 above)
Comment (ysampal) [reply templates/modules/b2b-account/quotes/quotes.hyper.live]: Done.

File: templates/modules/b2b-account/quotes/quotes.hyper.live (hunk 1 above)
Comment (ysampal) [reply templates/modules/b2b-account/quotes/quotes.hyper.live]: Done

**Comment by ghost** [comment]
There was a merge issue with this PR. I had to go back and amend this commit to add in some missing changes to integrate the custom select first added in COM-2880 (#296).

Basically, I changed:
- Fix the template for the quotes grid in quotes.hyper.live. That means the `filter-container` and the filters for account name, number, status, etc.
- Integrate the custom select component with the account name filter. This includes wiring up the select and events in quotes.js.

See the amended commit for the final code that was actually released.

**Pull request merged**